### ISSUE TITLE
Small text about image dimensions and its size would be very very handy

### ISSUE THREAD

**Issue by spajak** [d2phap/ImageGlass#524]
somewhere, small, next to the image:
`1200x800 | 65kB`
that's all

**Comment by Heacien** [comment]
I think that this would be unnecessary as all the details are in the window title.

![github](https://user-images.githubusercontent.com/12281340/57448338-6bef2000-7259-11e9-8c76-66607896c463.PNG)

**Comment by d2phap** [comment]
Hi @spajak 
You can find that information on title bar

![image](https://user-images.githubusercontent.com/3154213/57453839-fd7e8200-7299-11e9-8b94-c0e1f74892b4.png)

**Comment by spajak** [comment]
Yes, i know, but i use full screen all the time, and for every image i have to exit full screen mode to see the info. If you only need this info occasionally, thats fine, but for me it is as important as the image itself. This could be an option only for full screen. And its not very difficult to add it.

**Comment by d2phap** [comment]
That makes sense!
